cli/command/system/info.go

```diff
@@ -25,6 +25,7 @@ type infoOptions struct {
 type clientInfo struct {
 	Debug    bool
 	Context  string
+	Features map[string]string
```

Comment:
Can this be just `[]string`?

Reply:
Yes, that's something I'm very open to. I think there are several options and I'm happy to go with any:

1. `Features map[string]string` with each feature able to take a value, this could be something other than enabled for other features.
2. `Features []string` and just have each feature a name. Simple and too the point.
3. `Features []feature` where feature is just `type feature struct { name string }` today, but able to add more fields if needed in the future. This mirrors the plugin layout.
4. `Buildkit bool` just hardcode a single field for buildkit, and then need another for the next feature later. This has the advantage of being easy to template with `--format`.